src/Apps/Consign/Routes/SubmissionFlow/ContactInformation/ContactInformation.tsx

```diff
@@ -208,15 +208,16 @@ export const ContactInformation: React.FC<ContactInformationProps> = ({
   return (
     <>
       {isFirstStep && !artworkId ? (
+        // FIXME: Inappropriate component for this; don't replicate browser back button
         <TopContextBar
           displayBackArrow
           hideSeparator
           onClick={() => router.go(-1)}
```

Comment:
I think if you ever see this in our codebase it's probably a sign that something is wrong. We don't want to replicate the browser back button.